### PR TITLE
Plugin: Implement the elements API using React

### PR DESCRIPTION
**Pull request by youknowriad**
Just a small PR to raise the discussion on whether we should use `react` or something else for the WP Element API

### CODE REVIEW

**Comment by mtias** [comment]
We probably want to have a separate discussion about this, I like the `element` abstraction that gives us a bit more freedom. I'm ok with getting this in to start building.

File: external/@wordpress/elements/index.js [hunk 1]
```diff
@@ -10,7 +16,7 @@
  * @return {wp.Element}                  Element
  */
 export function createElement( type, props, ...children ) {
```
Comment (aduth) [inline external/@wordpress/elements/index.js]: Do you think it's better to have a clear abstraction layer, or should we just export them straight from React?

```js
export { createElement } from 'react';
export { render } from 'react-dom';
```

Comment (mimo84) [inline external/@wordpress/elements/index.js]: Another alternative is simply use aliases in webpack if in the future is decided to use a different library than React.

Comment (youknowriad) [inline external/@wordpress/elements/index.js]: Well! the current approach is more "verbose" but it's self-documenting the fact that it's an abstraction.

**Comment by youknowriad** [comment]
Merging this for now

**Pull request merged**